lighty-core/lighty-codecs-util/src/main/java/io/lighty/codecs/util/JsonNodeConverter.java

```diff
@@ -159,13 +163,18 @@ public Writer serializeRpc(final SchemaInferenceStack schemaInferenceStack,
     @SuppressWarnings({"checkstyle:illegalCatch"})
     public NormalizedNode deserialize(final SchemaInferenceStack schemaInferenceStack,
             final Reader inputData) throws DeserializationException {
-        final NormalizedNodeResult result = new NormalizedNodeResult();
+        final QName lastNodeQname = schemaInferenceStack.isEmpty() ? SchemaContext.NAME
```

Comment:
This has to be rebased on top of the correct SchemaInferenceStack usage.

Reply:
Done in #951